content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
    :align: center
    :alt: Enable the Google Calendar API.
 
-Create credentials
-------------------
+OAuth consent screen
+--------------------
 
-Now that you have created your API project and enabled the Google Calendar API, you need to create
-credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
-four steps to create your API credentials.
+Now that the API project has been created OAuth should be configured. Click on :guilabel:`OAuth
+consent` in the left menu and then select the :guilabel:`User Type`.
 
-- In the first step, :guilabel:`Credential Type`, select the :guilabel:`Google Calendar API` and
-  :guilabel:`User Data` options. Then, click :guilabel:`Next`.
+.. warning::
+   Personal Gmail Accounts are only allowed to be :guilabel:`External` User Type, which means
+   Google may require an approval or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
+   account allows for :guilabel:`Internal` User Type. It should also be noted that while the API
+   connection is in the :guilabel:`External` :guilabel:`Testing` mode then no approval is necessary
+   from Google. User limits in this :guilabel:`Testing` mode is set to 100 users.
```

Comment:
...Google may require an approval, or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
   account allows for :guilabel:`Internal` User Type. It should also be noted that, while the API
   connection is in the :guilabel:`External` :guilabel:`Testing` mode, then no approval is necessary
   from Google. User limits in this :guilabel:`Testing` mode is set to 100 users.